stestr/commands/load.py

```diff
@@ -180,7 +181,13 @@ def make_tests():
             case = testtools.DecorateTestCaseResult(case, decorate)
             yield (case, str(pos))
 
-    case = testtools.ConcurrentStreamTestSuite(make_tests)
+    if serial:
+        for _, stream in enumerate(streams):
+            # Calls StreamResult API.
+            case = subunit.ByteStreamToStreamResult(
+                stream, non_subunit_name='stdout')
```

Comment:
What happens if there is more than one stream we tell it to load? Doesn't this ignore all but the last stream?

Reply:
I'm thinking to get around that we probably should make like L195-L238 a separate function like `_load_case()` or something. Then inside the loop here and after L190 call it with the ConcurrentStreamTestSuite case we call that.

Reply:
for sure, I'll do that.

Reply:
well, I'm a bit confused. If we separate like that, what about L191-L194? I probably misunderstand something, though.. I guess we should make L19**1**-L238 a separate function to `_load_case()`?

Reply:
For this approach to to work we need to have L191-194 outside the loop (well really just L192 since that creates a new run in the repository, but they kind of all go together) Since the new function will be called from inside the loop we can't have the inserter be allocated inside the function. We'll have to move L191-194 above the loop and then pass the inserter object into `_load_case()`